### DISCUSSION TITLE
Firebase Server error: "Your API key is invalid, please check you have copied it correctly."

### DISCUSSION THREAD

**Discussion by nathanmartin5937** [nss-evening-cohort-14/evening-client-side#125] [Help Me!] [answered]
not sure how to get rid of these api key errors

```
t {code: "auth/invalid-api-key", message: "Your API key is invalid, please check you have copied it correctly.", a: null}
a: null
code: "auth/invalid-api-key"
message: "Your API key is invalid, please check you have copied it correctly."
__proto__: Error
constructor: ƒ t(a,b,c)
toJSON: ƒ d()
w: ƒ ()
__proto__: Object
```

**Comment by drteresavasquez** [comment, marked as answer]
This is identified in the Readme for the project. Make sure to read all the steps and follow them.

https://github.com/nss-nightclass-projects/IN-CLASS-almost-amazon-template#get-started

**Reply by BenJarrett** [reply]
So this is very similar to the issues we are having for Pan Am at the moment. 

 still getting an API key is invalid. We have looked at this over and over, Referred to the other tickets about invalid api keys #193 , which was ours and we fixed by taking out the commas in the .env file. But everything matches. Tried to serve and re serve like in #125 , We are going to double-check our Netlify environmental variables again.

Our environmental variables look correct. I'm not gonna lie, we are completely stuck on this.

We fixed our API key yesterday with this issue by removing commas.
I'm completely stuck atm


I'm just really not sure what else to try.

**Comment by nathanmartin5937** [comment]
Answer: close server and re serve it
